### DISCUSSION TITLE
One Time job based on a StartAt of another Timezone

### DISCUSSION THREAD

**Discussion by s3YwCf2ZbfJG4SHAfjQMAjtsf** [quartznet/quartznet#1897] [Q&A] [answered]
If I want to perform a one time job relative to a Time in another TimeZone, other than calculating the appropriate StartAt, is there anyway to tell Quartz the timezone?

```
ISimpleTrigger trigger = (ISimpleTrigger) TriggerBuilder.Create()
    .WithIdentity("trigger1", "group1")
    .StartAt(myStartTime) // some Date 
    .ForJob("job1", "group1") // identify job with name, group strings
    .Build();
```

With CronSchedule, you can indicate what timezone it is supposed to follow.

**Comment by lahma** [comment, marked as answer]
Simple trigger doesn't care about time zones, it only takes start time as UTC (I'm assuming trigger that doesn't repeat and thus we just want single fire). So basically you need to create a `DateTimeOffset` at source and then convert it to target time zone using `TimeZoneInfo`'s methods (TimeZoneInfo of target).
